src/ci/scripts/install-mingw.sh

```diff
@@ -39,10 +39,7 @@ if isWindows; then
     esac
 
     if [[ "${CUSTOM_MINGW:-0}" == 0 ]]; then
```

Comment:
I think this is just a dead code so a followup PR could get rid of all `CUSTOM_MINGW` occurrences.